tests/test_0017_multi_basket_multi_branch_fetch.py

```diff
@@ -310,6 +310,7 @@ def test_cache():
         skhep_testdata.data_path("uproot-sample-6.20.04-uncompressed.root"),
         object_cache=100,
         array_cache="100 MB",
+        handler=uproot.source.file.MemmapSource,
```

Comment:
Not so sure I am not breaking anything here, should the `fsspec` source behave exactly the same as the previous default source?

Reply:
If the default file-reader becomes non-memory mapping, that's okay. `MemmapSource` is still opt-in, and memory-mapping wasn't as important of a performance feature as it was originally thought to be. (The data have to be decompressed, byte-swapped, and concatenated, so the data in the file can't be used directly, anyway.)